tools/lsp/language/goto.rs

```diff
@@ -78,14 +78,22 @@ fn goto_node(node: &SyntaxNode) -> Option<GotoDefinitionResponse> {
     }]))
 }
 
-#[test]
-fn test_goto_definition() {
-    fn first_link(def: &GotoDefinitionResponse) -> &LocationLink {
-        let GotoDefinitionResponse::Link(link) = def else { panic!("not a single link {def:?}") };
-        link.first().unwrap()
-    }
+#[cfg(test)]
+mod tests {
```

Comment:
i'm not a fan of tests module

Reply:
I need a place where I can import `TextSize` without it being visible when not building the tests.

Otherwise CI will go red.

Reply:
you can import it in the function itself.